packages/zarf-registry/zarf.yaml

```diff
@@ -47,6 +47,11 @@ variables:
     description: Enable the Horizontal Pod Autoscaler for the registry
     default: "true"
 
+  - name: REGISTRY_CA_BUNDLE
+    description: Bundle of trusted certificates to mount into the registry container
+    default: ""
+    autoIndent: true
```

Comment:
```suggestion
    description: Filepath to a bundle of trusted certificates to mount into the registry container
    default: ""
    autoIndent: true
    type: file
```

After testing this a bit more locally everything looked pretty good!  One thing that _may_ make sense though (want to hear what this looks like in your use case) would be using the `type` field to pull in a file from a filename rather than a CA bundle as a multiline field.  This would provide the following benefits:

1. On super large CA bundles (an edge case but still possible) actions could fail due to exceeding the maximum size for environment variables (on Windows this is 32,767 characters)
2. It would allow using a CA bundle in the more familiar pem format rather than having to template it into a config file yourself manually.

Might not make sense if you aren't getting a file back in your use case, and requires an additional file to manage but may be worth it.  This would also require an added check to change line `194` of `src/pkg/utils/io.go` to prevent an erroneous warning on an empty string (which will never be a valid file)

```
				if template.Type == types.FileVariableType && value != "" {
```

Reply:
Just following up to see if you had thoughts on the above - I think it may be a bit cleaner but want to make sure that also makes sense for your use case.

Reply:
No problem at all, just wanted to make sure you had what you needed.

Reply:
Just checking back on this one - if needed we can open a new PR (maintaining your authorship) to help get this across the finish line (assuming you are good with the type: file suggestion)